framework/doc/content/source/partitioner/BlockWeightedPartitioner.md

```diff
@@ -0,0 +1,34 @@
+
+!template load file=stubs/moose_object.md.template name=BlockWeightedPartitioner syntax=/Mesh/Partitioner/BlockWeightedPartitioner
+
+## Overview
+
+Under multiphysics environment, some mesh blocks have more variables and more work load than others .  In parallel, the work
+is spread out based on partitioners that assign equal number of elements to each processor. This will causes imbalanced simulations.
+BlockWeightedPartitioner allow users to specify different weights for different blocks, e.g., low weights for light blocks
+and high weights for heavy blocks.  Usage:
```

Comment:
Let's add: "Weights should be proportional to the relative numbers of DOFs per element in each mesh block.  The weights do not need to be pre-normalized to any particular number." to this paragraph.

Reply:
But it is better to use small numbers. 

1 vs 2 is better than 1000 vs 2000

Reply:
Will add your words

Reply:
I mostly wanted to convey that they don't need to add up to any specific number.